reference/release-notes/1.22.1.md

```diff
@@ -0,0 +1,38 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.22.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* Support for Vulkan 1.3 for Android 13 images.<!--AC-2278-->
```

Comment:
As discussed over the MM, we postponed the [fix](https://bugs.launchpad.net/anbox-cloud/+bug/2065281) to 1.23.0. So let's drop it from 1.22.1 release announcement here.